tests/kernel/mem_heap/shared_multi_heap/boards/mps2_an521.overlay

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2021 Carlo Caione <ccaione@baylibre.com>
```

Comment:
nit: 2022? :)

Reply:
hahah yeah, but I'm definitely not going to push only to fix this ;)